lib/ansible/modules/packaging/os/redhat_subscription.py

```diff
@@ -234,16 +238,22 @@ def unsubscribe(self):
 
     def update_plugin_conf(self, plugin, enabled=True):
         plugin_conf = '/etc/yum/pluginconf.d/%s.conf' % plugin
+
```

Comment:
so instead of updating this here again, i would use the module_utils/redhat function now